lib/adapters/http/read.js

```diff
@@ -41,6 +48,19 @@ class Iterator {
             this.adapter.trigger('error', err);
         })
         .then(() => {
+            if (this.adapter.followLinkHeader && res.headers['link']) {
+                var parsed = parseLinkHeader(res.headers['link']);
+                if (parsed.next) {
+                    this.adapter.logger.debug('following link header', parsed.next.url);
+                    return this.adapter.fetch(parsed.next.url)
+                    .then((res) => {
+                        this.parse(res, from, to);
+                        res.resume();
```

Comment:
What exactly is the story with the pause/resuming? Do the tests fail if you take it out?